<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>60</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Schim</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../res/img/icon.svg</normaloff>../../res/img/icon.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabView">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="SheetView" name="tab">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="vimKeyStatus">
        <property name="text">
         <string>VIM key status (text changes dynamically)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSheet">
    <property name="title">
     <string>&amp;Sheet</string>
    </property>
    <addaction name="actionNewSheet"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionPrint"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndoInSheet"/>
    <addaction name="actionRedoInSheet"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSheet"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionInsertLine"/>
   <addaction name="actionInsertRect"/>
   <addaction name="actionInsertText"/>
  </widget>
  <widget class="QDockWidget" name="dockSymbols">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="todoButton">
       <property name="text">
        <string>todoButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ComponentList" name="componentList"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNewSheet">
   <property name="text">
    <string>New sheet</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionInsertLine">
   <property name="text">
    <string>Line</string>
   </property>
   <property name="toolTip">
    <string>Insert a line</string>
   </property>
  </action>
  <action name="actionUndoInSheet">
   <property name="text">
    <string>&amp;Undo in &amp;sheet</string>
   </property>
   <property name="toolTip">
    <string>Undo the last action in the current sheet</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedoInSheet">
   <property name="text">
    <string>&amp;Redo in &amp;sheet</string>
   </property>
   <property name="toolTip">
    <string>Redo in the current sheet</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>&amp;Save &amp;As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionInsertRect">
   <property name="text">
    <string>Rect</string>
   </property>
   <property name="toolTip">
    <string>Insert a rectangle</string>
   </property>
  </action>
  <action name="actionInsertText">
   <property name="text">
    <string>Text</string>
   </property>
   <property name="toolTip">
    <string>Insert a text object</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>&amp;Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SheetView</class>
   <extends>QWidget</extends>
   <header>ui/sheetview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ComponentList</class>
   <extends>QListWidget</extends>
   <header>ui/widgets/componentlist.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
